diff --git a/dist/runtime/plugin.client.mjs b/dist/runtime/plugin.client.mjs
index c1800ae6614c24045a0153bf538b5d9f64e84dbc..4386a1603f60a1196b1b9ccb68b8eef528795605 100644
--- a/dist/runtime/plugin.client.mjs
+++ b/dist/runtime/plugin.client.mjs
@@ -1,7 +1,5 @@
 import { defineNuxtPlugin } from "#app";
 export default defineNuxtPlugin(async (nuxtApp) => {
   const { io } = await import("socket.io-client");
-  const socket = io();
-  nuxtApp.provide("socket", socket);
   nuxtApp.provide("io", io);
 });
